### ISSUE TITLE
Improve the Hub admin UI to be … more like Medium

### ISSUE THREAD

**Issue by joesb** [wunderkraut/wunderhub#27]
Cf. http://www.wunderkraut.com/blog/a-medium-like-editor-for-drupal-8/2015-03-17

**Comment by lewisnyman** [comment]
This is a challenge, as the key feature of Medium is that the editing is done on the frontend of the site, so you can see how your changes would actually look.

As the hub only provides JSON, this would be a problem that would have to be solved on a site that consumes Hub content. We'd have to add a CRUD API to the Hub so consuming sites could write update content.

**Comment by steveparks** [comment]
I don't think it needs to actually happen on the frontend to meet this story... just that the editing experience needs to be more pleasurable in the backend - a nice big content area, easy to drop in images, create links etc.

**Comment by lewisnyman** [comment]
Removing the requirement for it to be like Medium, it just has to be good :)
